markdown/plext/using_pljava.html.md.erb

```diff
@@ -299,411 +331,770 @@ Scalar types are mapped in a straightforward way. This table lists the current m
+After obtaining a connection, you can prepare and execute statements similar to other JDBC connections. Refer to the `Users` class Java source code from the example above. 
+
+Limitations of the PL/Java JDBC driver include the following:
 
-- The transaction cannot be managed in any way. Thus, you cannot use methods on the connection such as:
+- A transaction cannot be managed in any way. As such, you cannot use the following methods on the JDBC connection:
    - `commit()`
    - `rollback()`
    - `setAutoCommit()`
    - `setTransactionIsolation()`
-- Savepoints are available with some restrictions. A savepoint cannot outlive the function in which it was set and it must be rolled back or released by that same function.
-- A ResultSet returned from `executeQuery()` are always `FETCH_FORWARD` and `CONCUR_READ_ONLY`.
-- Meta-data is only available in PL/Java 1.1 or higher.
+- Savepoints are available, with some restrictions. A savepoint cannot outlive the function in which it was set, and it must be rolled back or released by that same function.
+- A `ResultSet` returned from `executeQuery()` is always `FETCH_FORWARD` and `CONCUR_READ_ONLY`.
 - `CallableStatement` (for stored procedures) is not implemented.
-- The types `Clob` or `Blob` are not completely implemented, they need more work. The types `byte[]` and `String` can be used for `bytea` and `text` respectively.
+- The types `Clob` or `Blob` are not completely implemented. Use the types `byte[]` and `String` for `bytea` and `text`, respectively.
 
 ## <a id="exceptionhandling"></a>Exception Handling 
 
-You can catch and handle an exception in the HAWQ backend just like any other exception. The backend `ErrorData` structure is exposed as a property in a class called `org.postgresql.pljava.ServerException` (derived from `java.sql.SQLException`) and the Java try/catch mechanism is synchronized with the backend mechanism.
+You can catch and handle an exception in the HAWQ backend just like any other exception. The backend `ErrorData` structure is exposed as a property in the `org.postgresql.pljava.internal.ServerException` class (derived from `java.sql.SQLException`), and the Java `try/catch` construct is synchronized with the backend mechanism.
 
-**Important:** You will not be able to continue executing backend functions until your function has returned and the error has been propagated when the backend has generated an exception unless you have used a savepoint. When a savepoint is rolled back, the exceptional condition is reset and you can continue your execution.
+**Important:** If the backend has generated an exception and you have set a savepoint, the exception condition is reset when the savepoint is rolled back, allowing you to continue your execution.  If the backend has generated an exception and you have *not* used a savepoint, you will not be able to continue executing backend functions until your function has returned and the error has been propagated.
```

Comment:
Edit to last sentence:

"If the backend generates an exception and you have *not* used a savepoint, then you cannot execute backend functions until the function returns and propagates the error.